app/controllers/guides_controller.rb

```diff
@@ -147,8 +147,8 @@ def guide_params(with = {})
         :related_discussion_href,
         :related_discussion_title,
         :update_type,
+        :change_reason_for_change,
```

Comment:
Or just name it `editons.reason_for_change`? The `change_reason_for_change` is a bit of a mouthful.